routes/web.php

```diff
@@ -1,6 +1,36 @@
 <?php
-
+use App\Http\Controllers\RegisterController;
 use App\Http\Controllers\HomeController;
+use App\Http\Controllers\LoginController;
 use Illuminate\Support\Facades\Route;
+use App\Models\User;
+
+
+Route::get('tinker', function() {
+    Auth()->logout();
+});
+Route::middleware(['auth'])->group(function () {
+
+
+
+});
+
+
+Route::middleware(['guest'])->group(function () {
```

Comment:
Maak geen middleware voor guest. Zet al deze routes boven de Auth middleware. De guest middleware heeft toch geen functie en is alles dat standaard in web zit